extension/history.js

```diff
@@ -59,9 +60,9 @@ export class Database {
    * primaryKey), "next" is most recent last (based on primaryKey)
    * @returns {Promise} The documents
    */
-  getAll(tbName, direction) {
+  static getAll(dbName, tbName, direction) {
```

Comment:
It looks like this isn't supported by our tooling. I guess Firefox supports it now? It didn't when I started this but times change quickly!

But, since this makes the linter angry, either that would have to be resolved or this fixed. I'd recommend just rewriting these to `Database.get = function(dbName, tbName, primaryKey) {...};`